### DISCUSSION TITLE
Is this package maintained

### DISCUSSION THREAD

**Discussion by Arcalise08** [MapsterMapper/Mapster#713] [General]
9 months since last commit, 12 pull requests sitting with no updates for months. Is there an active maintainer for this package?

**Comment by andrerav** [comment]
Hi @Arcalise08, yes there is. The situation with the PR's currently in queue is a bit complicated, and due to some unforeseen circumstances in my private life I haven't had time to focus on this for a while. I've set aside time later this week to get this sorted and push some new pre-release packages to nuget. Thank you very much for your patience and understanding.

**Reply by Arcalise08** [reply]
Oh I completely understand that. Just wanted to make sure it wasn't going the way of the dodo.

Thanks for the reply and the effort!

**Comment by jgorski21** [comment]
Hey, what about this package? Will it be supported? There is a lot of good work in this project.
